### PR TITLE
Qualcomm_7-12-2021

### PR DESCRIPTION
**Pull request by QC-CNA**

### CODE REVIEW

File: 2021/1xxx/CVE-2021-1888.json [hunk 1]
```diff
@@ -1,18 +1,69 @@
 {
```
Comment (cve-team) [inline 2021/1xxx/CVE-2021-1888.json:1]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2021/1xxx/CVE-2021-1888.json:1]: ('Invalid control character at: line 43 column 24 (char 2815)',)

File: 2021/1xxx/CVE-2021-1890.json [hunk 2]
```diff
@@ -1,18 +1,69 @@
 {
```
Comment (cve-team) [inline 2021/1xxx/CVE-2021-1890.json:1]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2021/1xxx/CVE-2021-1890.json:1]: ('Invalid control character at: line 43 column 24 (char 2841)',)

File: 2021/1xxx/CVE-2021-1931.json [hunk 3]
```diff
@@ -1,18 +1,69 @@
 {
```
Comment (cve-team) [inline 2021/1xxx/CVE-2021-1931.json:1]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2021/1xxx/CVE-2021-1931.json:1]: ('Invalid control character at: line 43 column 24 (char 2469)',)

File: 2021/1xxx/CVE-2021-1965.json [hunk 4]
```diff
@@ -1,18 +1,69 @@
 {
```
Comment (cve-team) [inline 2021/1xxx/CVE-2021-1965.json:1]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2021/1xxx/CVE-2021-1965.json:1]: ('Invalid control character at: line 43 column 24 (char 2476)',)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2021/1xxx/CVE-2021-1888.json (hunk 1 above)
Comment (QC-CNA) [reply 2021/1xxx/CVE-2021-1888.json:1]: Updated Qualcomm_7-12-2021 branch. please check and let us know if any issues

Comment (QC-CNA) [reply 2021/1xxx/CVE-2021-1888.json:1]: Updated Qualcomm_7-12-2021 branch. please check and let us know if any issues

File: 2021/1xxx/CVE-2021-1890.json (hunk 2 above)
Comment (QC-CNA) [reply 2021/1xxx/CVE-2021-1890.json:1]: Updated Qualcomm_7-12-2021 branch. please check and let us know if any issues

File: 2021/1xxx/CVE-2021-1931.json (hunk 3 above)
Comment (QC-CNA) [reply 2021/1xxx/CVE-2021-1931.json:1]: Updated Qualcomm_7-12-2021 branch. please check and let us know if any issues

File: 2021/1xxx/CVE-2021-1965.json (hunk 4 above)
Comment (QC-CNA) [reply 2021/1xxx/CVE-2021-1965.json:1]: Updated Qualcomm_7-12-2021 branch. please check and let us know if any issues

Comment (QC-CNA) [reply 2021/1xxx/CVE-2021-1965.json:1]: Updated Qualcomm_7-12-2021 branch. please check and let us know if any issues

**Pull request merged**